scripts/setup_truefoundry_cluster.sh

```diff
@@ -0,0 +1,233 @@
+#!/bin/bash
+set -e
+
+# Configuration variables
+CONTROL_PLANE_URL="$1"
```

Comment:
Lets add validation for presence of these variables

Reply:
added the changes